### DISCUSSION TITLE
hello, I want to known capacitorjs 7 have what some break change, could you answer me?

### DISCUSSION THREAD

**Discussion by shynome** [ionic-team/capacitor#7718] [Q&A] [answered]
I don’t understand why CapacitorJS has so many major version breaks and provides so little explanation as to why.

**Comment by jcesarmobile** [comment, marked as answer]
Capacitor does a yearly major release, mostly to keep it up to date with latest Android SDK updates, which themselves are a breaking change.

We document the breaking changes every year, so not sure why you say we provide little explanation.

For Capacitor 7 the docs are not available yet because we just released the first alpha yesterday and it's mostly for our own plugins start using it. 

We will most likely have the docs ready for the first Capacitor 7 beta.

**Reply by shynome** [reply]
Thank you very much
